common/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ set(COMMON_SRC
         "${CMAKE_CURRENT_LIST_DIR}/decompress-huffman.h"
         "${CMAKE_CURRENT_LIST_DIR}/updates-model.h"
         "${CMAKE_CURRENT_LIST_DIR}/updates-model.cpp"
+        "${CMAKE_CURRENT_LIST_DIR}/cah-model.h"
```

Comment:
Should these really be in common/?

Reply:
I saw that the concept is making 99% of the UI code common, so I placed it there.
The sw-update files are there, output-model...
Don't you want AC trigger on DQT app?

Reply:
Not specified... is it there right now?

Reply:
Yes , the reset works but the trigger no because it needs RGB sensor.
I am not sure how easy it will be to remove it, almost all the code is common.
it will require dummy functions I believe.
What do you think?

Reply:
For now it's OK, but keep it in mind as we move forward that we need to move out of the DQT.